flutter_local_notifications/lib/src/platform_specifics/android/notification_details.dart

```diff
@@ -251,4 +252,20 @@ class AndroidNotificationDetails {
   /// For a list of a values, refer to the documented constants prefixed with "FLAG_" (without the quotes) at https://developer.android.com/reference/android/app/Notification.html#constants_1.
   /// For example, use a value of 4 to allow the audio to repeat as documented at https://developer.android.com/reference/android/app/Notification.html#FLAG_INSISTEN
   final Int32List additionalFlags;
+
+  /// This provides some additional information that is displayed in the
```

Comment:
Sorry missed this earlier but this plugin is following the effective dart documentation guidelines and currently the way this is done violates [this rule]( https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph) and [this rule](https://dart.dev/guides/language/effective-dart/documentation#prefer-starting-function-or-method-comments-with-third-person-verbs). Whilst the latter is a preference, it has been used for this plugin (e.g. "Specifies...")

Could you look at this changing the documentation to meet the guidelines and what's been done in the plugin so far? I know this is currently lifted from the official Android documentation.

Reply:
No problem, consistency is important.  I updated the PR, please check it again.